### ISSUE TITLE
`mc config host list` truncates secret-key to 40 characters

### ISSUE THREAD

**Issue by donatello** [minio/mc#2176]
## Expected behaviour
The secret should be printed out completely, or at least indicate that it is truncated with trailing dots or so.

## Actual behaviour
Secret key is truncated to 40 characters - so when copy-pasted and used, it doesn't work. This leads the user to believe their credentials are wrong, leading to confusion.

## Steps to reproduce the behaviour
Set a long secret key (e.g. minio gateway mode for azure; azure uses very long secret keys), and run the above command.

## mc version
- latest as of this writing.

**Comment by vadmeste** [comment]
The same problem exists with endpoints urls and access keys. I think we should avoid formatting those fields and only keep formatting the first field (alias)

```
        local : http://localhost:9000
      myminio : http://192.168.1.100:9000 | minio | minio123 | S3v4
 myminio-dc-1 : https://172.19.0.2:9000 | minio | minio123 | S3v4
myminio-dc-10 : http://172.19.0.11:9000 | minio | minio123 | S3v2
myminio-dc-11 : http://172.19.0.12:9000 | minio | minio123 | S3v2
 myminio-dc-5 : http://172.19.0.6:9000 | minio | minio123 | S3v2
 myminio-dc-6 : http://172.19.0.7:9000 | minio | minio123 | S3v2
 myminio-dc-7 : http://172.19.0.8:9000 | minio | minio123 | S3v2
 myminio-dc-8 : http://172.19.0.9:9000 | minio | minio123 | S3v2
 myminio-dc-9 : http://172.19.0.10:9000 | minio | minio123 | S3v2
myminio-nginx : http://localhost | access_key | secret_key | S3v4
         play : https://play.minio.io:9000 | Q3AM3UQ867SPQQA43P2F | zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG | S3v4
     playanon : https://play.minio.io:9000
         riak : http://localhost:80 | D34X2FNDMIRLATS-SMMO | 4AWiuY_Z3trGrG_eDm7oh0L1hsUoMC_XTRntEQ== | S3v2
```

**Comment by harshavardhana** [comment]
Fixed by #2180
